webui/react/src/hooks/useMetricNames.ts

```diff
@@ -27,6 +29,7 @@ const useMetricNames = (
       setMetrics(Loaded([]));
       return;
     }
+    if (!isEqual(actualExpIds, previousExpIds)) setMetrics(NotLoaded);
```

Comment:
Currently, when the `ExpIds` change the metrics are still set as `Loaded` which causes us to show the `No Data To Plot` message since metrics may have not been loaded for the newly selected experiments.